assets/css/style.css

```diff
@@ -185,7 +185,7 @@ body {
 
 /* === 3.1 HOME BODY === */
 
-body.home {
+/*body.home {
```

Comment:
@jarmstrng I commented it out because I thought it made more sense to do that than override all these settings in custom.css, but I didn't delete because I didn't want to alter style.css. In the original design, there was nothing below the countdown so the css fixing the screen was on .body.home.static. Which is why since we are doing single page app I thought it made more sense to put the countdown into it's own section and put styles on that section instead of the body.